### PR TITLE
Include blog post for release-1.5

### PR DESCRIPTION
**Pull request by dprotaso**
Fixes: https://github.com/knative/docs/issues/4976#issuecomment-1146856965

### CODE REVIEW

File: blog/docs/releases/announcing-knative-v1-5-release.txt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+Follow the instructions in [Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+## Table of Contents
+- [Highlights](#highlights)
+- [Serving](#serving)
+- [Eventing](#eventing)
+- [Eventing Extensions](#eventing-extensions)
+    - [Apache Kafka Broker](#apache-kafka-broker)
+    - [RabbitMQ Broker and Source](#rabbitmq-broker-and-source)
+- `kn` [CLI](#client)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-5-release.txt:25]:
```suggestion
- [`kn` CLI](#client)
```

Comment (psschwei) [reply blog/docs/releases/announcing-knative-v1-5-release.txt:25]: (looks like the link has always been on just the CLI part, but in a vacuum I'd say it's better to link the whole line)

File: blog/docs/releases/announcing-knative-v1-5-release.txt [hunk 2]
```diff
@@ -0,0 +1,205 @@
+---
+title: "1.5 release"
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-5-release.txt:2]: This doc also needs to be added to the blog navigation: https://github.com/knative/docs/blob/main/blog/config/nav.yml#L16

Comment (psschwei) [reply blog/docs/releases/announcing-knative-v1-5-release.txt:2]: Doc should also be named `...release.md`

File: blog/docs/releases/announcing-knative-v1-5-release.txt [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+linkTitle: "1.5 release"
+Author: "[dprotaso](https://twitter.com/lintinmybelly)"
+Author handle: https://github.com/dprotaso
+date: 06-06-2022
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-5-release.txt:6]:
```suggestion
date: 2022-06-06
```

**Review by psschwei — CHANGES_REQUESTED** [review]
some nits, mostly mkdocs related

File: blog/docs/releases/announcing-knative-v1-5-release.txt (hunk 3 above)
Comment (dprotaso) [reply blog/docs/releases/announcing-knative-v1-5-release.txt]: I prefer YYYY-MM-DD

**Comment by psschwei** [comment]
Think other than the lint fix, this one's good to go

**Comment by csantanapr** [comment]
Thank you @dprotaso 🎉

**Pull request merged**